tests/storage/mailbox_test.py

```diff
@@ -364,8 +364,17 @@ def reply_msg_callback(vol_data):
 
         times = [end[k] - start[k] for k in start]
         times.sort()
-        log.info("stats: messages=%d delay=%.3f best=%.3f worst=%.3f avg=%.3f",
-                 messages, delay, times[0], times[-1], sum(times) / len(times))
+        best = times[0]
+        worst = times[-1]
+        average = sum(times) / len(times)
+
+        log.info(
+            "stats: messages=%d delay=%.3f best=%.3f average=%.3f worst=%.3f",
+            messages, delay, best, average, worst)
+
+        assert best < 4 * EVENT_INTERVAL
+        assert average < 7 * EVENT_INTERVAL
```

Comment:
Fails in github:

    >       assert average < 7 * EVENT_INTERVAL
    E       assert 0.7567226762539666 < (7 * 0.1)

We need larger timeouts.

Reply:
This shows that these kind of tests are quite problematic. It would be good to add a comment where these numbers (4, 7, 10) comes from or what does it mean if this test fails.

Reply:
Yes this test is problematic. If we don't test that the numbers makes sense, we can have performance regressions. If we test, the test will be flaky - when running on slow or overloaded machine the test will fail.

I think I will a comment showing how to run the tests and inspect the "stats" log, and make the asserts more forgiving for slow environment.